e2e/tests/stepDefinitions/manage.ts

```diff
@@ -28,10 +29,17 @@ Given('I create a premises with all necessary details', () => {
 
   page.getLocalAuthorityAreaIdByLabel('North Lanarkshire', 'localAuthorityAreaId')
 
-  cy.get('@localAuthorityAreaId').then(localAuthorityAreaId => {
+  page.getCharacteristicIdByLabel('Park nearby', 'parkNearbyCharacteristicId')
+  page.getCharacteristicIdByLabel('Floor level access', 'floorLevelAccessCharacteristicId')
+
+  cy.then(function _() {
```

Comment:
Why the underscore? keeping the linter happy?

Reply:
Basically 😭

Can't use arrow notation here, as then you lose access to the function's `this` object, which I'm using to get the aliased values in a tidier way than lots of nested `cy.get('@somealias').then()` calls. And there's the convention of using `_` as a prefix for function param names that are unused but needed to satisfy a signature, or for unused variables when destructuring a tuple